src/Compilers/Core/MSBuildTask/MSBuildTask.csproj

```diff
@@ -11,6 +11,7 @@
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetFrameworks>$(RoslynPortableTargetFrameworks46)</TargetFrameworks>
     <DisableImplicitFrameworkReferences>false</DisableImplicitFrameworkReferences>
+    <NoWarn>$(NoWarn);CA1819</NoWarn> <!-- CA1819 (Properties should not return arrays) disabled as it is very common across this project. -->
```

Comment:
Nit: Please put the comment above property. That's the general style we use in build files.